ngraph/test/type_prop/convolution.cpp

```diff
@@ -2659,7 +2659,8 @@ TEST(type_prop, conv_v1_partial_data_shape_dynamic)
     auto conv = make_shared<op::v1::Convolution>(
         data_batch, filters, strides, pads_begin, pads_end, dilations, auto_pad);
 
-    ASSERT_TRUE(conv->get_output_partial_shape(0).same_scheme({PartialShape::dynamic()}));
+    ASSERT_TRUE(conv->get_output_partial_shape(0).same_scheme(
```

Comment:
you should use ASSER_EQUAL to fully check each dimension

Reply:
To my understanding, [`same_scheme`],(https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/src/partial_shape.cpp#L234) loops shape dimensions and checks if they are equal.